### PR TITLE
docs: hander -> authenticator

### PR DESCRIPTION
**Pull request by kenjis**
Ref #67

First, remove **handler** from the documentation.

### CODE REVIEW

File: docs/2 - authentication.md [hunk 1]
```diff
@@ -99,7 +100,7 @@ if($result->isOK()) {
 If the attempt fails a `failedLoginAttempt` event is triggered with the credentials array as 
 the only parameter. Whether or not they pass, a login attempt is recorded in the `auth_logins` table.
 
-If `allowRemembering` is `true` in the `Auth` config file, you can tell the Session Handler
+If `allowRemembering` is `true` in the `Auth` config file, you can tell the Session Authenticator
```
Comment (MGatner) [inline docs/2 - authentication.md:103]: I don't care about the casing but it should be consistent. 
```suggestion
If `allowRemembering` is `true` in the `Auth` config file, you can tell the Session authenticator
```

File: docs/2 - authentication.md [hunk 2]
```diff
@@ -282,8 +283,8 @@ if ($user->tokenCant('forums.manage')) {
 ## Controller Filters
 
 Shield provides 3 [Controller Filters](https://codeigniter.com/user_guide/incoming/filters.html) you can 
-use to protect your routes, `session`, `tokens`, and `chained`. The first two cover the two handlers, 
-`Session` and `AccessTokens`, respectively. The `chained` filter will check both handlers in sequence 
+use to protect your routes, `session`, `tokens`, and `chained`. The first two cover the two authenticators,
+`Session` and `AccessTokens`, respectively. The `chained` filter will check both authenticators in sequence
```
Comment (MGatner) [inline docs/2 - authentication.md:287]:
```suggestion
use to protect your routes, `session`, `tokens`, and `chained`. The first two cover the `Session` and `AccessTokens` authenticators, respectively. The `chained` filter will check both authenticators in sequence
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Definite improvement. A couple small suggestions.

**Comment by kenjis** [comment]
@MGatner Thanks!

**Review by lonnie-vault — APPROVED** [review]

**Pull request merged**